hack/tools/go.mod

```diff
@@ -5,74 +5,220 @@ go 1.19
 replace sigs.k8s.io/cluster-api => sigs.k8s.io/cluster-api v1.4.8
 
 require (
+	github.com/aquasecurity/trivy v0.47.0
 	github.com/drone/envsubst/v2 v2.0.0-20210730161058-179042472c46
 	github.com/golang/mock v1.6.0
 	github.com/golangci/golangci-lint v1.52.2
 	github.com/itchyny/gojq v0.12.9
 	github.com/joelanford/go-apidiff v0.5.0
-	github.com/onsi/ginkgo/v2 v2.9.2
+	github.com/onsi/ginkgo/v2 v2.9.4
+	golang.org/x/vuln v1.0.1
 	gotest.tools/gotestsum v1.10.0
-	k8s.io/code-generator v0.26.10
+	k8s.io/code-generator v0.27.3
```

Comment:
but we have `v0.26.10` version used go.mod file in the root dir

Reply:
I missed this, will fix.

Reply:
added it to replace section (pin to v0.26.10)